torch/testing/_internal/common_utils.py

```diff
@@ -904,6 +904,12 @@ def _check_module_exists(name: str) -> bool:
 # as we had before.  By default, we don't run these tests.
 TEST_WITH_CROSSREF = os.getenv('PYTORCH_TEST_WITH_CROSSREF', '0') == '1'
 
+
+if TEST_CUDA and 'PARALLEL_TESTING' in os.environ:
+    # we usually run 3 processes and other libraries take up about 11% of space -> .33 - .11 = .22
+    torch.cuda.set_per_process_memory_fraction(0.22)
```

Comment:
Well, let take the NUM_PROCS constant into this calculation, i.e. `(1 - 0.11) / NUM_PROCS`